cura/CuraApplication.py

```diff
@@ -1457,7 +1458,11 @@ def updatePlatformActivity(self, node = None):
             self._scene_bounding_box = scene_bounding_box
             self.sceneBoundingBoxChanged.emit()
 
-        self._platform_activity = True if count > 0 else False
+        if count > 0:
+            ProjectOutputDevice.setLastOutputName(None)
```

Comment:
I'm a bit surprised to find it here. Wouldn't it be better in `deleteAll` ?

Reply:
I had it in delete all first, but that doesn't cover the ctrl-A, delete (select all, delete) use-case. Since that only indirectly deletes all (doesn't go via that function) since you only happen to have all selected.

Reply:
Ok, I could argue that making a `ctrl-A + delete` doesn't have the same meaning than making a "clear build plate" but I agree that in the current workflow, most users would expect that.

Reply:
If you where to argue that I can could counter that with that what we _actually_ want here is not checking for 'delete all', but 'are there any meshes on the buildplate that might be a reference to the previously opened project/thing', which would be true when there is at least on mesh still there. 😉 

(Also 'activity' might be a bit of a misnomer here, as far as I can see, the 'activity' is just there being anything on the buildplate ... but we can't really rename that function.)

Reply:
It took me some time to figure out the meaning of this function indeed. But as you can, we can't really rename it.

And your other argument is also very true, so we can definitely keep this behavior.